keyboards/kinesis/keymaps/soft_dev/keymap.c

```diff
@@ -0,0 +1,148 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _BASE,   // default layer
+    _KEYPAD, // arrow keys, shifted symbols, HOME, END, PGUP, PGDN, FKeys, CTRL(TAB)
+    _NUMPAD  // numpad keys
+};
+
+/* ,------------------------------------------------------------------------------------------------------------------------.
+* | Esc | F1 | F2 | F3 | F4 | F5 | F6 | F7 | F8 |                           | F9 | F10 | F11 | F12 |PSCR|SLCK|PAUS|KPAD|BOOT|
+* |--------+------+------+------+-------+------+--------------------------------+------+------+------+------+------+--------|
+* |   -_   |  1!  |  2@  |  3#  |  4$   |  5%  |                                |  6^  |  7&  |  8*  |  9(  |  0)  |  =+    |
+* |--------+------+------+------+-------+------|                                +------+------+------+------+------+--------|
+* | Tab    |   Q  |   W  |   E  |   R   |   T  |                                |   Y  |   U  |   I  |   O  |   P  | \|     |
+* |--------+------+------+------+-------+------|                                |------+------+------+------+------+--------|
+* | Del    |   A  |   S  |   D  |   F   |   G  |                                |   H  |   J  |   K  |   L  |  ;:  | '"     |
+* |--------+------+------+------+-------+------|                                |------+------+------+------+------+--------|
+* | Shift  |   Z  |   X  |   C  |   V   |   B  |                                |   N  |   M  |  ,.  |  .>  |  /?  |Sht/Ent |
+* `--------+------+------+------+-------+-------                                `------+------+------+------+------+--------'
+*          | `~   | Gui  | Esc  |MO(Num)|                                              | [{   |  ]}  |      |      |
+*          `----------------------------'                                              `---------------------------'
+*                                        ,---------------.         ,---------------.
+*                                        |Alt+Tab|  Alt  |         |  Alt  |TO(Num)|
+*                                 ,------|------ |-------|         |-------+-------+------.
+*                                 |      |       | Shift |         | Shift |       |      |
+*                                 | BkSp | KPAD  |-------|         |------ | KPAD  | Space|
+*                                 |      |       | Ctrl  |         | Ctrl  |       |      |
+*                                 `----------------------'         `----------------------'
+*/
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_BASE] = LAYOUT_pretty(
+KC_ESC,KC_F1,KC_F2,KC_F3,KC_F4,KC_F5,KC_F6,KC_F7,KC_F8,                 KC_F9,KC_F10,KC_F11,KC_F12,KC_PSCR,KC_SCRL,KC_PAUS,TO(_KEYPAD),QK_BOOT,
```

Comment:
This is a nitpick, so feel free to ignore, but this bothers me.

Specifically, most/all keycodes have a 7 character name/alias, so you can have a 9 wide column (eg `1234567, `), so you can have consistent formatting/placement.  So this scrunched together could be better spaced.